package.json

```
@@ -8,28 +8,28 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.28",
     "@fortawesome/free-brands-svg-icons": "^5.13.0",
     "@fortawesome/free-solid-svg-icons": "^5.13.0",
-    "@fortawesome/react-fontawesome": "0.1.9",
+    "@fortawesome/react-fontawesome": "^0.1.10",
```

I think we should lock to a specific version (not use ^ or ~) as long as major version is zero.
According to https://semver.org/:
_Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable._
```suggestion
    "@fortawesome/react-fontawesome": "0.1.10",
```

---

Fixed